gdal/ogr/ogrct.cpp

```diff
@@ -960,6 +960,66 @@ OCTClone(OGRCoordinateTransformationH hTransform)
         OGRCoordinateTransformation::FromHandle(hTransform)->Clone());
 }
 
+/************************************************************************/
+/*                             OCTGetSourceCS()                         */
+/************************************************************************/
+
+/**
+ * Transformation's source coordinate system reference.
+ *
+ * This is the same as the C++ function OGRCreateCoordinateTransformation::GetSourceCS
```

Comment:
It should be mentioned that the returned object might be null, and is owned by hTransform and should not be freed.